lib/TPP/Dialect/Tpp/BufferizableOpInterfaceImpl.cpp

```diff
@@ -191,13 +235,17 @@ static bool bufferizesToMemoryWriteBinaryImpl(Operation *op,
   // If the rhs can bufferize in place with the result return true.
   if (opOperand.getOperandNumber() == 1 &&
       opOperand.get().getType() == op->getResult(0).getType() &&
-      !isConstantVal(opOperand.get()))
+      !isConstantVal(opOperand.get())) {
     return true;
+  }
+
   // If the lhs can bufferize in place with the result return true. Note that
   // if both can bufferize with the result we select the rhs first.
```

Comment:
Does it make any difference whether we choose lhs or rhs?

Reply:
No, but we need to pick one. If both are fine we always pick the rhs.

Reply:
Ah okay I see the need for it now 👍